jni/src/test/java/com/horizen/vrfnative/VRFKeyPairTest.java

```diff
@@ -1,127 +1,127 @@
-            FieldElement message = FieldElement.deserialize(messageBytes);
-            VRFProof proof = VRFProof.deserialize(proofBytes);
-            FieldElement expectedVrfOutput = FieldElement.deserialize(vrfOutputBytes)
-        )
-        {
-            assertNotNull("sk deserialization must not fail", sk);
-            assertNotNull("message deserialization must not fail", message);
-            assertNotNull("proof deserialization must not fail", proof);
-            assertNotNull("expectedVrfOutput deserialization must not fail", sk);
-
-            try(FieldElement vrfOutput = keyPair.getPublicKey().proofToHash(proof, message))
-            {
-                assertNotNull("VRF Proof verification and VRF Output computation has failed.", vrfOutput);
-
-                // Check vrfOutput == expectedVrfOutput
-                assertEquals("vrfOutput and expectedVrfOutput must be equal", vrfOutput, expectedVrfOutput);
-            }
-        }
-    }
-
-    @Test
-    public void testRandomProveVerify() throws Exception {
-        int samples = 100;
-
-        for(int i = 0; i < samples; i++) {
-            try
-            (
-                VRFKeyPair keyPair = VRFKeyPair.generate();
-                FieldElement fieldElement = FieldElement.createRandom();
-                FieldElement wrongFieldElement = FieldElement.createRandom()
-            )
-            {
-                assertNotNull("Key pair generation was unsuccessful.", keyPair);
-                assertTrue("Public key verification failed.", keyPair.getPublicKey().verifyKey());
-
-                try
-                (
-                    VRFProveResult proofVRFOutputPair = keyPair.prove(fieldElement);
-                    FieldElement vrfOutput = keyPair.getPublicKey().proofToHash(proofVRFOutputPair.getVRFProof(), fieldElement)
-                )
-                {
-                    assertNotNull("Attempt to create vrf proof and output failed.", proofVRFOutputPair);
-                    assertNotNull("VRF Proof verification and VRF Output computation must not fail.", vrfOutput);
-                    assertEquals("prove() and proof_to_hash() vrf outputs must be equal", proofVRFOutputPair.getVRFOutput(), vrfOutput);
-                    assertNull("VRF Proof verification must fail", keyPair.getPublicKey().proofToHash(proofVRFOutputPair.getVRFProof(), wrongFieldElement));
-                }
-            }
-        }
-    }
-}
+//package com.horizen.vrfnative;
+//
+//import com.horizen.librustsidechains.FieldElement;
+//import com.horizen.schnorrnative.SchnorrKeyPair;
+//import com.horizen.schnorrnative.SchnorrSignature;
+//import org.junit.Test;
+//
+//import static org.junit.Assert.assertNull;
+//import static org.junit.Assert.assertNotNull;
+//import static org.junit.Assert.assertTrue;
+//import static org.junit.Assert.assertEquals;
+//
+//public class VRFKeyPairTest {
+//
+//
+//    @Test
+//    public void testGenerate() throws Exception {
```

Comment:
Don't see any usage of hardcoded data here. Maybe we can keep it ?

Reply:
Yes, this test is working, I've uncommented it